History.md

```diff
@@ -1,3 +1,7 @@
+### 2.3.1 - *July 20 2017*
+
+  * Fix [https://github.com/web-animations/web-animations-js/issues/157](missing web-animations.min.js issue)
```

Comment:
Yep, it looks like I didn't actually push to origin dev (not sure if I was meant to).
@ewilligers will be pushing that commit to origin dev soon so this diff should disappear.